lib/init.coffee

```diff
@@ -10,4 +14,44 @@ module.exports =
         type: 'string'
 
   activate: ->
-    console.log 'activate linter-gjslint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-gjslint.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'gjslint'
+      grammarScopes: ['source.js', 'source.js.jquery', 'text.html.basic']
+      scope: 'file'
+      lintOnFly: true
+      lint: (editor) =>
+        filePath = editor.getPath()
+        cwd = path.dirname(filePath)
+        tempFile path.basename(filePath), editor.getText(), (tmpFilePath) =>
+          params = [
+            '--nobeep',
+            '--quiet',
+            tmpFilePath
+          ]
+          return helpers.exec(@executablePath, params, {cwd}).then (stdout) ->
+            regex = /^Line\s(\d*)\, E\:(\d*):\s(.*)$/
+            lines = stdout.split('\n').filter (line) ->
+              line.indexOf('Line') == 0
+
+            return [] unless lines.length
+            return lines.map (msg) ->
+              res = regex.exec(msg)
+              [all, line, code, text] = res
+              line = parseInt(line, 10)
+              return {
+                text: text,
+                type: 'error',
+                filePath: filePath,
+                range: [
+                  [line - 1, 0],
```

Comment:
You might think about using `helpers.rangeFromLineNumber` to give a range that covers the entire content of the line, since `gjslint` doesn't give column numbers.

(Just using the first column is fine, it just means you get no inline markers and when a user clicks on your message the inline tooltip, if enabled, shows at the start of the line instead of where the code is.)